tools/logrotate_config/package_after_install_script.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Create the log file for gcsfuse package.
+mkdir -p /var/log/gcsfuse/
+
+# Installs logrotate configuration files.
+mkdir -p /etc/logrotate.d/
+
+cat > /etc/logrotate.d/gcsfuse <<EOF
+/var/log/gcsfuse/*.log {
+    rotate ${LOGROTATE_FILES_MAX_COUNT:-5}
+    copytruncate
```

Comment:
can you add comments on what these are doing.

Reply:
I'll create a log-rotation docs to explain all the log-rotation related informations useful for user.